app/src/main/java/org/kinecosystem/kinit/viewmodel/spend/TransferActivityModel.kt

```diff
@@ -41,21 +42,23 @@ class TransferActivityModel(private val app: EcosystemApp, var transferActions:
         if (app.transferData != null) {
             val started = transferManager.startTransferRequestActivity(activity, app.identifier, app.transferData.launchActivityFullPath)
             if (!started) {
-                transferActions?.onConnectionError()
+                transferActions?.onStartConnectingError()
+                analytics.logEvent(Events.Business.CrossAppKinFailure("", Analytics.FAILURE_TYPE_ERROR))
```

Comment:
error message: "Could not launch "+app.transfer.Data.launchActivityFullPath+" on "+app.identifier